### ISSUE TITLE
Amiodarone produces worsening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9896]
**Preferred term label:**
Amiodarone produces worsening

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Myxedema

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as

Exacerbated by amiodarone exposure
HP:0034962
